src/unstract/sdk/tool/validator.py

```diff
@@ -123,15 +123,14 @@ def _validate_file_size(self, input_file: Path) -> None:
         self.tool.stream_log(
             f"Checking input file size... (max file size: {max_file_size})"
         )
-        file_size = input_file.stat().st_size
-        self.tool.stream_log(
-            f"Input file size: {self._human_readable_size(file_size)}"
-        )
+        if self.tool.workflow_filestorage:
+            file_size = self.tool.workflow_filestorage.size(path=input_file)
+        else:
+            file_size = input_file.stat().st_size
```

Comment:
When does this code under else get triggered? Everywhere w ewill be expecting a file storage instance going forward right?